### PR TITLE
4330 remove fraud language from classes and copy

### PR DESCRIPTION
**Pull request by avinhurry**
## Context

Removing the use of the word fraud from classes and content. 

We have agreed with the policy team that the language of "fraud" is not helpful when considering duplicate applications.

## Changes proposed in this pull request

- Rename `fraud_match` to `duplicate_match`
- Rename `UpdateFraudMatchesWorker` to `UpdateDuplicateMatchesWorker`
- Fix tests
- Delete `remove_access_form` and test as this is no longer required in the new duplicate matching implementation

## Guidance to review

- Have a play around with the duplicate matching service
- Create new matches, and run the worker, check that everything works as expected
- Block/Unblock submission and lock/unlock candidate accounts, check that everything still works as expected

## Link to Trello card

https://trello.com/c/tYvHDvBL/4330-remove-fraud-language-from-classes-and-copy

### CODE REVIEW

File: app/models/duplicate_match.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
-class FraudMatch < ApplicationRecord
+class DuplicateMatch < ApplicationRecord
+  self.table_name = 'fraud_matches'
```
Comment (frankieroberto) [inline app/models/duplicate_match.rb:2]: Is the plan to rename the table in a future PR, or leave it as it is?

**Review by frankieroberto — APPROVED** [review]

File: app/models/duplicate_match.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/models/duplicate_match.rb:2]: Leave as it is. Otherwise will be a massive work just to change a one-liner.

Comment (duncanjbrown) [reply app/models/duplicate_match.rb:2]: Agreed!

Comment (duncanjbrown) [reply app/models/duplicate_match.rb:2]: Maybe add a comment?

Comment (frankieroberto) [reply app/models/duplicate_match.rb:2]: Yeah I was just curious! 😄

File: app/models/candidate.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Candidate < ApplicationRecord
   has_many :application_choices, through: :application_forms
   has_many :application_references, through: :application_forms
   belongs_to :course_from_find, class_name: 'Course', optional: true
-  belongs_to :fraud_match, optional: true
+  belongs_to :duplicate_match, class_name: 'DuplicateMatch', foreign_key: 'fraud_match_id', optional: true
```
[the remark this answers is not in the document]
Comment (tomas-stefano) [reply app/models/candidate.rb]: Yes it can. Removed.

**Pull request merged**